app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
```

Comment:
Would a `PageController` or `StaticPageController` not be more precise/specific?